services/question-service/src/generate-mongodb-schema.ts

```diff
@@ -0,0 +1,113 @@
+import "dotenv/config";
+import { MongoClient, ServerApiVersion } from 'mongodb';
+import { exit } from "process";
+
+const uri = process.env.MONGO_ATLAS_URL || 'mongodb://localhost:27017';
+
+// Create a MongoClient with a MongoClientOptions object to set the Stable API version
+const mongoClient = new MongoClient(uri, {
+  serverApi: {
+    version: ServerApiVersion.v1,
+    strict: true,
+    deprecationErrors: true,
+  }
+});
```

Comment:
looks like we're not using Prisma for this?

Reply:
Nope. But we can change this to use prisma later on

Reply:
ok feel free to merge after fixing conflics